testing/scenario_app/lib/main.dart

```diff
@@ -53,32 +22,39 @@ void main() {
     ..onMetricsChanged = _onMetricsChanged
     ..onPointerDataPacket = _onPointerDataPacket
     ..scheduleFrame();
+
   final ByteData data = ByteData(1);
   data.setUint8(0, 1);
   window.sendPlatformMessage('waiting_for_status', data, null);
 }
 
+void _handleDriverMessage(Map<String, dynamic> call) {
+  switch (call['method']) {
+    case 'set_scenario':
+      final Map<String, dynamic> scenarioData = call['args'];
+      loadScenario(scenarioData);
+      print('Loading scenario ${scenarioData['name']}');
+    break;
+    default:
+      print('Unimplemented method: ${call['method']}.');
```

Comment:
Shouldn't this throw? This seems like a good way for a test to pass even though it should fail because of e.g. a typo.